fe/lib/components/JobCategorySelect/queries.ts

```diff
@@ -0,0 +1,66 @@
+import { gql } from 'apollo-boost';
+
+const JOB_CATEGORY_ATTRIBUTES = gql`
+  fragment jobCategoryAttributes on JobCategory {
+    id {
+      value
+    }
+    name
+  }
+`;
+
+const NESTED_JOB_CATEGORY_ATTRIBUTES = gql`
+  fragment nestedJobCategoryAttributes on JobCategory {
+    ...jobCategoryAttributes
+    parent {
+      ...jobCategoryAttributes
+    }
+    children {
+      ...jobCategoryAttributes
+      parent {
+        ...jobCategoryAttributes
+      }
+    }
+  }
+  ${JOB_CATEGORY_ATTRIBUTES}
+`;
+
+export const JOB_CATEGORY = gql`
+  query($id: String!) {
+    jobCategory(id: $id) {
+      ...nestedJobCategoryAttributes
+    }
+  }
+  ${NESTED_JOB_CATEGORY_ATTRIBUTES}
+`;
+
+export const JOB_CATEGORIES = gql`
+  query($schemeId: String!) {
+    jobCategories(schemeId: $schemeId) {
+      ...nestedJobCategoryAttributes
+    }
+  }
+  ${NESTED_JOB_CATEGORY_ATTRIBUTES}
+`;
+
+export const JOB_CATEGORY_SUGGESTION = gql`
+  query(
+    $positionProfile: JobCategorySuggestionPositionProfileInput!
+    $schemeId: String!
+    $hirerId: String
```

Comment:
I was waiting for this PR so I could make someone else do my work!

Can we move this `hirerId` in to `positionProfile.positionOrganizations` as per SEEK-Jobs/indie-graphql-server#596

Reply:
Yeah, sure thing! Once this is merged back into our internal repo's I'll remove their usage of the deprecated structure.

Reply:
I've just removed it from the variables for this query. I don't actually use this query yet just copied the whole lot for now. My next PR will add JobCategoriesSuggest which will, I'll use the updated `JobCategorySuggestionPositionProfileInput` shape with hirerid there.